python/strings_udf/strings_udf/__init__.py

```diff
@@ -59,13 +64,33 @@ def compiler_from_ptx_file(path):
                 "files. Please file an issue reporting this error and how you "
                 "installed cudf and strings_udf."
             )
-        sms = [
-            os.path.basename(f).rstrip(".ptx").lstrip("shim_") for f in files
-        ]
-        selected_sm = max(sm for sm in sms if sm < cc)
-        ptxpath = os.path.join(
-            os.path.dirname(__file__), f"shim_{selected_sm}.ptx"
-        )
+
+        virtual_sms = []
```

Comment:
We don't need multiple lists. We want a table setup that looks like:

```
[
"90a" -> [ <path_to>/shim_90a-real.ptx> ],
"90" -> [ <path_to>/shim_90.ptx>, <path_to>/shim_90-virtual.ptx>, ...],
"70" -> [ <path_to>/shim_70.ptx> ],
]

If we have a `dev.compute_capability+a` entry or `dev.compute_capability` entry we use that, otherwise we use the next numeric lowest without the `a` postfix.